openplacereviews-boot/src/main/resources/bootstrap/opr-0-web-bot.json

```diff
@@ -27,6 +27,11 @@
       			"key" : "tourism",
       			"values" : ["hotel","motel","hostel","apartment","guest_house"],
             	"type" : ["node", "way"]
+          },
+          "attraction": {
+            "key": "historic",
+            "values": ["ruins"],
```

Comment:
probably we should add much more attraction list + tourism. we can double check what's supported by OsmAnd tourism category

Reply:
tourism=camp_site and tourism=caravan_site would be nice

Reply:
Also `historic=castle`, `historic=aqueduct`, `historic=church` etc.